inf-clojure.el

```diff
@@ -189,6 +189,15 @@ number (e.g. (\"localhost\" . 5555))."
    (stringp (car x))
    (numberp (cdr x))))
 
+(defcustom inf-clojure-skip-project-detection nil
```

Comment:
Basically this should be `inf-clojure-project-type` and the docstring should say it's purpose is to override the default project type resolution via `.dir_locals.el`.